packages/ui-tooltip/src/Tooltip/props.ts

```diff
@@ -55,6 +55,10 @@ type TooltipRenderChildren = (
 ) => React.ReactNode
 
 type TooltipOwnProps = {
+  /**
+   * provides a reference to the underlying html root element
+   */
+  elementRef?: (element: Element | null) => void
```

Comment:
Please add new props to the `allowedProps` array too

Reply:
done